packages/core/test/lib/utils/traceData.test.ts

```diff
@@ -115,15 +103,15 @@ describe('getTracingMetaTags', () => {
             traceFlags: TRACE_FLAG_SAMPLED,
           };
         },
-        transaction: undefined,
       },
       mockedScope,
       undefined,
     );
 
     expect(tags).toEqual({
-      sentryTrace: '<meta name="sentry-trace" content="12345678901234567890123456789012-1234567890123456-1"/>',
+      'sentry-trace': '12345678901234567890123456789012-1234567890123456-1',
     });
+    expect('baggage' in tags).toBe(false);
```

Comment:
l: due to above's suggestion
```suggestion
    expect('baggage' in traceData).toBe(false);
```